.github/workflows/autoversion.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           if ! git diff --exit-code; then
             git config --global user.email "github-actions[bot]@users.noreply.github.com"
             git config --global user.name "github-actions[bot]"
-            git commit -am   "Auto-versioned $(git diff --name-only)"
+            git commit -am   "Auto-versioned: $(git diff --name-only | paste -s -d ' ' -)"
```

Comment:
`tr '\n' ' '` !!

Reply:
I did that in my first cut.

`tr` turns trailing newlines into trailing spaces.